data.sql

```diff
@@ -4,3 +4,10 @@ INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES('Gabumon', TO_DATE('15/11/2018', 'DD/MM/YYYY'), 2, true, 8);
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES('Pikachu', TO_DATE('07/01/2021', 'DD/MM/YYYY'), 1, false, 15.04);
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES('Devimon', TO_DATE('12/05/2017', 'DD/MM/YYYY'), 5, true, 11);
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES('Charmander', TO_DATE('08/02/2020', 'DD/MM/YYYY'), 0, false, 11);
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES('Plantmon', TO_DATE('15/11/2022', 'DD/MM/YYYY'), 2, true, 5.7);
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES('Squirtle', TO_DATE('02/04/1993', 'DD/MM/YYYY'), 3, false, 12.13);
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES('Angemon', TO_DATE('12/06/2005', 'DD/MM/YYYY'), 1, true, 45);
```

Comment:
Please change `weight_kg` values for `Charmander`, `Plantmon`, `Squirtle`, and `Angemon` to negative as can be seen in the requirements.